client/ayon_xstudio/plugins/load/open_in_xstudio.py

```diff
@@ -110,13 +112,106 @@ def load(
             filepath,
         ]
 
+        # TODO(plp): Is there a get_ayon_env() somewhere ?
+        ayon_env = dict(os.environ)
+        _set_ocio_env_var(context, ayon_env)
+
         try:
             # Run xStudio with these commands
-            run_detached_process(cmd)
+            run_detached_process(cmd, env=ayon_env)
             # Keep process in memory for some time
             time.sleep(0.1)
 
         except FileNotFoundError:
             self.log.error(  # noqa: TRY400
                 'File "%s" was not found.', os.path.basename(filepath)
             )
+
+
+def find_ayon_ocio_config(ocio_path: Path) -> Union[Path, None]:
+    """Find the AYON OCIO config path.
+
+    Args:
+        ocio_path (Path): The path to the OCIO config.
+
+    Returns:
+        Union[Path, None]: The path to the AYON OCIO config if found, None
+            otherwise.
+    """
+    log.debug("representation ocio: %s", ocio_path.as_posix())
+    path = Path(ocio_path)
+
+    if "OpenColorIOConfigs" not in path.parts:
+        return None
+
+    try:
+        from ayon_ocio import get_ocio_config_path
+
+    except ImportError:
+        pass
+    else:
+        ocio_folder = get_ocio_config_path()
+        log.debug("ocio root: %s", ocio_folder)
+        folder_index = path.parts.index("OpenColorIOConfigs")
+        server_ocio = Path(ocio_folder).joinpath(
+            *path.parts[folder_index + 1 :]
+        )
+        log.debug("server_ocio = %s", server_ocio)
+        if server_ocio.exists():
+            return server_ocio
+
+        log.debug("server_ocio doesn't exist !")
+
+    return None
+
+
+def _set_ocio_env_var(context: Dict[str, Any], env: dict) -> None:
+    """Set the OCIO environment variable based on the given context.
+
+    Args:
+        context (dict): The context to use for setting the OCIO environment
+            variable.
+        env (dict): The environment variables to update.
+
+    Raises:
+        KeyError: If the 'representation' key is not found in the context.
+    """
+    representation: dict = context.get("representation", {})
+    if not representation:
+        err = "Couldn't find 'representation' in context !"
+        raise KeyError(err)
+
+    colorspace_data: dict = representation.get("data", {}).get(
+        "colorspaceData", {}
+    )
+    if not colorspace_data:
+        log.warning("Couldn't find 'colorspaceData' in representation.")
+        log.warning("Not configuring OCIO !")
+        return
+
+    ocio_path = colorspace_data.get("config", {}).get("path")
+    if not ocio_path:
+        log.warning("Representation OCIO path undefined")
+        log.warning("Not configuring OCIO !")
+        return
+
+    ocio_path = Path(ocio_path)
+    if ocio_path.is_absolute():
+        ocio_path = ocio_path.resolve()
+
+    if not ocio_path.exists():
+        anatomy = Anatomy(context["project"]["name"])
```

Comment:
You can pass in the project from context
```suggestion
        anatomy = Anatomy(
            context["project"]["name"],
            project_entity=context["project"],
        )
```